sdk/eventhub/event-hubs/samples/javascript/README.md

```diff
@@ -18,8 +18,7 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | [useWithIotHub.js][usewithiothub]                                         | demonstrates how to use the `EventHubClient` with an `IotHub` instance                                                                                                                  |
 | [websockets.js][websockets]                                               | demonstrates how to use WebSockets enable Event Hubs to work over an HTTP proxy and in environments where the standard AMQP port 5671 is blocked (see the sample for more information)  |
 | [usingAadAuth.js][usingaadauth]                                           | demonstrates how to instantiate EventHubsClient using AAD token credentials obtained from using Service Principal Secrets                                                               |
-| [receiveEventsUsingCheckpointStore.js][receiveeventsusingcheckpointstore] | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance (requires running the `sendEvents` sample first) |
-| [receiveEvents.js][receiveevents]                                         | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance, as well as checkpointing along the way.         |
+| [receiveEvents.js][receiveevents]                                         | demonstrates how to use the EventHubConsumerClient to process events from all partitions of a consumer group in an Event Hubs instance.                                                 |
```

Comment:
Is there any reason we can't leave the link in and just point to the version in the eventhub-checkpointstore-blob package's sample folder? It would make it more easily discoverable.

Reply:
Thats a great idea

Reply:
I added a link and a short explanation below the table. All throughout the samples everything else in these tables has the property of "anything in the table is in the zip file," and I didn't want to break that consistency so I made it a little more visually distinct. I think we are good to merge here.